### DISCUSSION TITLE
I want to build ios only iphone

### DISCUSSION THREAD

**Discussion by dongwooklee96** [fastlane/fastlane#20198] [General]
Hi,
I want to build ios only iphone build not ipad how can I set this setting?
do you have any idea of this?

**Comment by BarbaraBryant** [comment]
The simulator is good, but it has limitations. For example, you can’t test push notifications with the simulator, you need an actual device.

Definitely worth getting a device. Maybe an SE on min storage. Should be able to pick up a second hand one pretty easily. [landstaronline](https://www.landstaronline.me/)

**Comment by pschneider** [comment]
@dongwooklee96 
If you mean that you don't want that your app is able to run on iPad at all, you can adjust your App target settings. Under "General" there is a section "Deployment Info" where you can untick the `iPad` checkbox.
